test/test-applications/integrations/Samples.Kafka/Properties/launchSettings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "profiles": {
+    "Samples.Kafka": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "DD_DOTNET_TRACER_HOME": "C:\\repos\\dd-trace-dotnet\\src\\bin\\tracer-home",
+        "COR_ENABLE_PROFILING": "1",
+        "CORECLR_ENABLE_PROFILING": "1",
+        "COR_PROFILER_PATH": "C:\\repos\\dd-trace-dotnet\\src\\bin\\tracer-home\\win-x64\\Datadog.Trace.ClrProfiler.Native.dll",
+        "DD_DUMP_ILREWRITE_ENABLED": "1",
+        "CORECLR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
+        "DD_TRACE_CALLTARGET_ENABLED": "true",
+        "DD_INTEGRATIONS": "C:\\repos\\dd-trace-dotnet\\src\\bin\\tracer-home\\integrations.json",
+        "DD_ENV": "andrew",
```

Comment:
Hello, `andrew`

Reply:
removed in https://github.com/DataDog/dd-trace-dotnet/pull/1444/commits/13c2a27450c0accdcb58f61a77ea84f5f859df08